.github/workflows/test-changed-auth.yml

```diff
@@ -49,27 +48,22 @@ jobs:
 
     runs-on: ubuntu-20.04
 
-    # Pin Chrome version 114.0.5735.90-1 to avoid install failures like "No such object: chromedriver/LATEST_RELEASE_115.0.5790".
-    # The failure happens because https://chromedriver.chromium.org/downloads only goes up to version 114.
-    # TODO(b/297380444) Update script to install the latest Chrome and ChromeDriver.
     steps:
       - name: install Firefox stable
         run: |
           sudo apt-get update
           sudo apt-get install firefox
           sudo apt-get install wget
-          sudo wget http://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_114.0.5735.90-1_amd64.deb
-          sudo apt-get install -f ./google-chrome-stable_114.0.5735.90-1_amd64.deb --allow-downgrades
-
+          npx @puppeteer/browsers install chrome@stable
```

Comment:
Just out of curiosity, why does the Firefox job install Chrome stable? Will something break if Chrome isn't there?

Reply:
I haven't checked this iteration, but yes, it was required previously. I can take it out and see what happens.